.gitignore

```diff
@@ -9,6 +9,8 @@
 # Helm chart dependencies
 deploy/helm/sumologic/charts
 deploy/helm/sumologic/charts/*
+deploy/helm/sumologic/tmpcharts
+deploy/helm/sumologic/tmpcharts/*
```

Comment:
Not sure. Why do we do `*` in addition to directory path?

Reply:
I don't think it's needed actually. `.gitignore` docs say:

> The pattern foo/ will match a directory foo and paths underneath it, but will not match a regular file or a symbolic link foo (this is consistent with the way how pathspec works in general in Git)

So I'll remove the wildcards.